spec/lib/deployers_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'spec_helper'
+require 'deployers'
+require 'timecop'
+require 'climate_control'
+
+RSpec.describe Deployers do
+  around do |ex|
+    ClimateControl.modify DEPLOYERS: 'one,two,three,four,five,six,seven' do
+      ex.run
+    end
+  end
+
+  describe '.for_today' do
+    it 'always returns the same value on the same day' do
+      Timecop.travel(2019,1,1) do
+        deployers_for_today = Array.new(10) { Deployers.for_today }
+        expect(deployers_for_today.uniq.count).to eq 1
```

Comment:
I'm not sure I follow what is being asserted here; so it's checking if `["five", "six", "four", "two", "seven", "three", "one"].uniq.count` is 1?... That seems wrong...

Reply:
`Array.new(10) { Deployers.for_today }` will return an array of 10 elements each populated by a fresh call to `Deployers.for_today`, so it should be true that this array contains only one unique element

Reply:
Ah yes!